GitHubClient.py

```diff
@@ -0,0 +1,355 @@
+import os
+import requests
+import json
+
+class GitHubClient(object):
+    """Basic client for getting the last diff and managing issues."""
+    existing_issues = []
+    milestones = []
+
+    def __init__(self):
+        self.github_url = os.getenv('INPUT_GITHUB_URL')
+        if not self.github_url:
+            raise EnvironmentError
```

Comment:
Error out early if GitHub-specific environment variable is not set. The exception is caught in `main`, where it used to fall back to other options.